PCSX2/pcsx2
# Use native language GameDB entries

### Description of Changes
Switches the main display of GameDB entries to the game's native language in Qt (bigpicture coming later)
(Note: I only added native language entries to Japanese games, we still need someone to do it for the Korean games.)
...plus a few modifications to the gamedb validation scripts so you actually see line numbers when validation fails

### Questions:
Should I include an option to force English titles?  If so, where should I put it?

### Rationale behind Changes
It would be pretty goofy to have the entire UI translated into Japanese, but still use English titles

### Suggested Testing Steps
- Make sure your Japanese games show in Japanese in the game list
- Make sure the Japanese games sort properly (should be gojuuon order by pronunciation of the kanji)
